framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -5732,6 +5733,10 @@ VkResult VulkanReplayConsumerBase::OverrideCreateRayTracingPipelinesKHR(
     }
     else
     {
+        GFXRECON_LOG_ERROR_ONCE(
+            "The replayed application used vkCreateRayTracingPipelinesKHR, which may require the "
+            "rayTracingPipelineShaderGroupHandleCaptureReplay feature for accurate capture and replay. The replay "
+            "device does not support this feature, so replay may fail.");
```

Comment:
Add error info for that the device doesn't support rayTracingPipelineShaderGroupHandleCaptureReplay in replaying CreateRayTracingPipelinesKHR.